rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -0,0 +1,70 @@
+---
+setup:
+  - do:
+      indices.create:
```

Comment:
Because creating an index with `constant_keyword` field on old version node makes shards unassigned and then make the cluster red, so I suggest combing the two test cases `Mappings` and `Supported queries` into one, set only one `skip-version`, and create index inside the test case so that we'll not create index against old version node when running `MixedClusterClientYamlTestSuiteIT`.